src/test/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+jdbc.driverName=org.h2.Driver
+jdbc.url=jdbc:h2:file:./src/test/resources/h2/sample
+jdbc.username=sa
+jdbc.password=
+
+marklogic.host=localhost
```

Comment:
I can guess, but I imagine most users will wonder - I see how the jdbc properties are copied into ApplicationProperties - but who uses the marklogic.* properties?